service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/cmo/DraftOrdersApprovedEventHandler.java

```diff
@@ -124,7 +125,7 @@ public void sendNotificationToCafcass(final DraftOrdersApproved event) {
 
             notificationService.sendEmail(
                     JUDGE_APPROVES_DRAFT_ORDERS,
-                    recipientIsWelsh.get().getEmail(),
+                    recipientIsWelsh.orElseThrow().getEmail(),
```

Comment:
We should probably add a specific error message or exception you want to throw here too